sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -125,6 +125,79 @@ def test_roc_curve_display_plotting(
     assert display.ax_.get_xlabel() == expected_xlabel
 
 
+@pytest.mark.parametrize("plot_chance_level", [True, False])
+@pytest.mark.parametrize(
+    "chance_level_kwargs",
+    [
+        {"linewidth": 1},
+        {"color": "b", "label": "DummyEstimator"},
+        {"color": "g", "linestyle": ":"},
```

Comment:
Here I really want only 2 parametrization:

```python
[None, {"linewidth": 1, color:"red", label="DummyEstimator"}]
```

It will simplify the test because we either have to check the default style or the given style. So below:

```python
if plot_chance_level and chance_level_kwargs is None:
    assert ...  # check the default style
elif plot_chance_level and chance_level_kwargs is not None:
    assert ...  # check that we have the provided style
else:
    assert ...  # we should not have any line